packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -517,18 +517,39 @@ export async function generateSummary(
 }
 
 // Check if the current-state matches that of the output-state.
-// Note operatorState subsumes the keys of the outputState
-export const isOperatorStateInSync = (operatorState: Record<string, any>, outputState: Record<string, any>) => {
+//
+// Note operatorState subsumes the keys of the outputState, thus if
+// all of outputState[x] matches operatorState[x] it is considered
+// to be in sync.
+export const isOperatorStateInSync = (
```

Comment:
This doesn't need to be a `const` as it is self contain. Keeping it a `function` will be easier to read than an arrow function, and to manipulate in a potential refactoring.

Reply:
Can you provide some examples of why arrow functions are harder to refactor or manipulate?

Reply:
Because you have to check that they have no side-effect, as a function does not have this issue.

Reply:
@YohannParis If we have a particular preference we want to try to adhere to as a team I am all on board with that. But this is not a valid argument, I can easily have side effect regardless of whether I use arrow-functions or the classic function declaration. 

```
const list = []

export const doSomething = () => {
  list.push(123)
}

export function doSomething() {
  list.push(123)
}
```

These two are functionally (excuse the pun) equivalent, they both have side effects of modifying another data structure that is not in their own respective closure.